docs/mms_on_fargate.md

```diff
@@ -0,0 +1,279 @@
+
+![architecture](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/architecture_2.png)
+
+Do not be worried if something is not clear from the picture. We are going to walk you through step by step. And these steps are:
+
+1. Familiarize yourself with MMS containers
+2. Create a SqueezeNet task definition (with the docker container of MMS) 
+3. Create Fargate ECS cluster
+4. Create Application Load Balancer
+5. Create Squeezenet Fargate service on the cluster
+6. Profit!
+
+Let the show begin...
+
+## Familiarize Yourself With Our Containers 
+
+With the current release of [MMS, 0.3](https://github.com/awslabs/mxnet-model-server/releases/tag/v0.3.0), we now providing official containers are provided with MMS preinstalled. There are 2 containers:
+
+* [awsdeeplearningteam/mms_cpu](https://hub.docker.com/r/awsdeeplearningteam/mms_cpu/)
+* [awsdeeplearningteam/mms_gpu](https://hub.docker.com/r/awsdeeplearningteam/mms_gpu/)
+
+In our article we are going to use cpu container (mms_cpu). 
+
+There are several constraints that one should consider when using Fargate:
+
+1. There is no GPU support at the moment.
+2. mms_cpu container is optimized for the Skylake Intel processors (that we have on our [C5 EC2 instances](https://aws.amazon.com/ec2/instance-types/c5/)). However, since we are using Fargate, unfortunately there is no guarantee that the actual hardware will be Skylake.
+
+Our containers come with preinstalled config of the SqueezeNet model (such a nice coincidence that we have decided to run inference for exactly this model :) ). Even though the config is pre-baked in the container, it is highly recommended to have a quick look at it. [Familiarize yourself](https://github.com/awslabs/mxnet-model-server/blob/master/docker/mms_app_cpu.conf),since for your own model, most likely you will have to update it. here is the line in the config that is pointing to the binary of the model:
+
+```
+https://github.com/awslabs/mxnet-model-server/blob/master/docker/mms_app_cpu.conf#L3
+```
+
+Looking closely, one can see that it is just a public HTTPS link to the binary:
+
+```
+https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+```
+
+So there is no need to pre-bake actual binary of the model to the container. You can just specify the HTTPS link to the binary.
+
+The last question that we need to address: how we should be starting our MMS within our container. And the answer is very simple, you just need to set the following [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint): 
+
+```bash
+mxnet-model-server start --mms-config /mxnet-model-server/mms_app_cpu.conf
+```
+
+And this it, nothing else.
+
+At this point, you are ready to start creating actual task definition.
+
+## Create a SqueezeNet Task Definition
+
+This is the first task where you finally ready to start doing something:
```

Comment:
My last comment stands... not sure what we're doing here.

I think what is missing is just a quick intro on how you need to wrap the container as a task for ECS to use.

Reply:
Sounds good, I will add a section for this